CHANGELOG.md

```diff
@@ -37,7 +37,11 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## Unreleased
 
-* nothing
+### Improvements:
+* Migrate `AddMarkerProposal` to v1 message based gov proposal [PR 1321](https://github.com/provenance-io/provenance/pull/1321)
+
+### Client Breaking
+* Migrate `AddMarkerProposal` to v1 message based gov proposal [PR 1321](https://github.com/provenance-io/provenance/pull/1321)
```

Comment:
When fixing the conflicts here, make sure to add an empty line after the heading and before the entries.